WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListViewModel.kt

```diff
@@ -83,5 +83,9 @@ class NotificationsListViewModel @Inject constructor(
 
     sealed class InlineActionEvent {
         data class SharePostButtonTapped(val notification: PostNotification): InlineActionEvent()
+
+        companion object {
+            val KEY_INLINE_ACTION = "inline_action"
```

Comment:
Changing this to const should fix the detekt issue

```suggestion
            const val KEY_INLINE_ACTION = "inline_action"
```